app/cafe.py

```diff
@@ -0,0 +1,23 @@
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError, NotWearingMaskError)
+import datetime
```

Comment:
import datetime should be before app.errors
And each error should be on separate line for better readability